packages/shared/graphql/generated/GetTestRunRecordings.ts

```diff
@@ -103,7 +103,9 @@ export interface GetTestRunRecordings_node_Workspace {
   testRuns: GetTestRunRecordings_node_Workspace_testRuns | null;
 }
 
-export type GetTestRunRecordings_node = GetTestRunRecordings_node_Recording | GetTestRunRecordings_node_Workspace;
+export type GetTestRunRecordings_node =
+  | GetTestRunRecordings_node_Recording
+  | GetTestRunRecordings_node_Workspace;
```

Comment:
It's strange but I think it's right. The `node` member in our schema can be either workspaces or recordings so when it generates the types, it doesn't know at this level which you're using.

You can see the same thing in other places using `node` e.g:

```
export type GetWorkspaceApiKeys_node = GetWorkspaceApiKeys_node_Recording | GetWorkspaceApiKeys_node_Workspace;
```

Reply:
But when would you ever want to treat a _recording_ and a _workspace_ as the same thing? They're not interchangeable.

Reply:
I guess it's a deficiency in the type generator? I'd have expected it to be able to figure out that with this graphql:
```
  query GetTestRunRecordings($workspaceId: ID!, $id: String!) {
    node(id: $workspaceId) {
      ... on Workspace {
```
it will only be returning `Workspace` types but apparently it can't.